### DISCUSSION TITLE
quasar dev doesn't work

### DISCUSSION THREAD

**Discussion by Godfadatun** [quasarframework/quasar#9035] [CLI - BEX mode]
So I have been going through a problem for starting up my code. Here is the error I currently get 

`(node:6682) UnhandledPromiseRejectionWarning:   Error: ENOENT: no such file or directory, open '/Users/danieladegoke/Desktop/r  epos/vue/paylidateFrontEnd/paylidate/src/router/index.js'
  
  - legacy-validations.js:104 legacyValidations
    [lib]/[quasar-cli]/lib/legacy-validations.js:104:16
  
  - quasar-config.js:655 QuasarConfig.compile
    [lib]/[quasar-cli]/lib/quasar-config.js:655:5
  
  - quasar-dev:156 goLive
    [lib]/[quasar-cli]/bin/quasar-dev:156:16
  
  - task_queues.js:93 processTicksAndRejections
    internal/process/task_queues.js:93:5
`

<img width="1438" alt="image" src="https://user-images.githubusercontent.com/36653089/116083098-23261100-a694-11eb-803b-3e2737381639.png">

**Comment by rstoenescu** [comment]
What version of "@quasar/app" are you using? `lib/legacy-validations.js` does not exist in "@quasar/app" v2 or v3. It has been removed long time ago...
